src/Strathweb.AspNetCore.AzureBlobFileProvider/IContainerFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Strathweb.AspNetCore.AzureBlobFileProvider
+{
+    public interface IContainerFactory
```

Comment:
sorry for the delay 😢
can we please rename it to `IBlobContainerFactory`? same for the default implementation

Reply:
No problem, and thanks for the feedback!  I've made the renames and pushed.